### PR TITLE
replace hyperkube binary with a bash script

### PR DESCRIPTION
**Pull request by p0lyn0mial**
since `hyperkube` has been removed from k/k (1.17) repository this pulls replaces it with a custom script that has similar behavior.

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
requires https://github.com/openshift/release/pull/6312

File: hack/build-go.sh [hunk 1]
```diff
@@ -14,11 +14,12 @@ platform="$(os::build::host_platform)"
...
-  if [[ "${platform}" == linux/* ]]; then
-    build_targets=(vendor/k8s.io/kubernetes/cmd/hyperkube)
-  else
-    build_targets=(vendor/k8s.io/kubernetes/cmd/hyperkube)
-  fi
+    build_targets=(vendor/k8s.io/kubernetes/cmd/kube-apiserver
+    vendor/k8s.io/kubernetes/cmd/kube-controller-manager
+    vendor/k8s.io/kubernetes/cmd/kube-proxy
+    vendor/k8s.io/kubernetes/cmd/kube-scheduler
+    vendor/k8s.io/kubernetes/cmd/kubectl
```
Comment (sttts) [inline hack/build-go.sh]: is kubectl used anywhere?

File: hack/build-go.sh [hunk 2]
```diff
@@ -14,11 +14,12 @@ platform="$(os::build::host_platform)"
...
 build_targets=("$@")
 if [[ -z "$@" ]]; then
-  if [[ "${platform}" == linux/* ]]; then
-    build_targets=(vendor/k8s.io/kubernetes/cmd/hyperkube)
-  else
-    build_targets=(vendor/k8s.io/kubernetes/cmd/hyperkube)
-  fi
+    build_targets=(vendor/k8s.io/kubernetes/cmd/kube-apiserver
+    vendor/k8s.io/kubernetes/cmd/kube-controller-manager
+    vendor/k8s.io/kubernetes/cmd/kube-proxy
```
Comment (p0lyn0mial) [inline hack/build-go.sh]: this needs to be removed as well.

**Comment by p0lyn0mial** [comment]
GCP is broken `Error setting IAM policy for project`

**Pull request merged**